bluemira/equilibria/shapes.py

```diff
@@ -119,49 +120,6 @@ def flux_surface_kuiroukidis(
     n_power: int = 8,
     n_points: int = 100,
 ):
```

Comment:
Could we have a small docstring here even though technically its allowed without by flake, just to direct users to the below function that gives the explanation. I know its only used internally here and not aimed to be exposed but its a reasonably complicated function and because python its still importable.

```suggestion
):
"""
Kuiroukidis flux surface individual quadrants

please see :func:flux_surface_kuiroukidis for more information
"""
```